macros/src/main/scala/com/avast/cactus/CactusMacros.scala

```diff
@@ -66,16 +67,16 @@ object CactusMacros {
     // unpack the implicit ClassTag tree
     val gpbSymbol = extractSymbolFromClassTag(c)(gpbCt)
 
-    val variableName = getVariable(c)
-
+    val variable = getVariable(c)
+    val variableName = variable.symbol.asTerm.fullName.split('.').last
```

Comment:
It does. Even empty string (which couldn't be present here, it's at least something like `gpb`) will have last, because the `split` never returns an empty array:
```
"".split('.').last == ""
```